Doc/library/pdb.rst

```diff
@@ -211,6 +211,9 @@ access further features, you have to do this yourself:
    The *readrc* argument defaults to true and controls whether Pdb will load
    .pdbrc files from the filesystem.
 
+   The *commands* argument, if given, would be a list of commands to execute
+   when the debugger starts. It has similar effects to the :file:`.pdbrc` file.
```

Comment:
The phrase “similar effects” is kind of vague. I am not sure how to say it more precisely. Link to the `.pdbrc` documentation? Not sure.